src/structured-data/schema.js

```diff
@@ -6,11 +6,16 @@ export function getSchemaType(type) {
   return { "@type": type };
 }
 
-export function getSchemaPerson(name) {
-  return Object.assign({}, getSchemaType("Person"), {
-    givenName: name,
-    name: name,
-  });
+export function getSchemaPerson(name, url = "") {
+  return Object.assign(
```

Comment:
why no spread?

Reply:
Currently, Spread Operator throws SyntaxError while using Babel.  we can use `@babel/plugin-transform-spread` plugin but for now, following the current way.